src/utils/localStorage.js

```diff
@@ -0,0 +1,25 @@
+const keyAccountConfirmed = (accountId, networkId) => `wallet.account:${accountId}:${networkId}:confirmed`
```

Comment:
I think's it's a bit misleading to put this into `utils/localStorage.js`, as it is specific to storing wallet accounts and other stuff touching `localStorage` isn't using this module. I feel like it makes sense to keep these in `wallet.js`.

Reply:
Done. My idea was to avoid circular dependencies. But code is simpler and it's working fine.